package.json

```diff
@@ -93,6 +96,7 @@
     "moment": "^2.13.0",
     "morgan": "^1.7.0",
     "node-sass": "^3.6.0",
+    "nsp": "^2.4.0",
```

Comment:
here is the tldr of the above comment :) @markolson maybe move this to dev dependencies since I don't think we need this on deploy